### PR TITLE
Adds invariant culture in SerializeOther method

### PR DESCRIPTION
**Pull request by tanukidarou**
Avoid conflicts with culture when converting string to float or double

### CODE REVIEW

File: Assets/Plugins/Appboy/Utilities/MiniJson.cs [hunk 1]
```diff
@@ -598,25 +598,23 @@ private void SerializeString(string str)
                 this.builder.Append('\"');
             }
 
-            private void SerializeOther(object value)
-            {
-                if (value is float
-                    || value is int
+            void SerializeOther(object value) {
+                if (value is float) {
```
[the remark this answers is not in the document]
Comment (tanukidarou) [reply Assets/Plugins/Appboy/Utilities/MiniJson.cs:602]: Ideally, it's best to cast the same type. In an updated version of MiniJSON library ([link](https://gist.github.com/darktable/1411710)) I see that they do the same thing.

[the remark this answers is not in the document]
Comment (Bucimis) [reply Assets/Plugins/Appboy/Utilities/MiniJson.cs:602]: Gotcha, not advocating for a change in how things would be casted, just how the if's are ordered. It's not too important though. We'll test this change out and release it in a future version of our SDK. Definitely appreciate the contribution!

**Comment by bryanlogan** [comment]
It looks like we might also need to add the Invariant to ParseNumber() as well.

How did you encounter this issue?  I'm trying to create a test case, but can't seem to create one that fails before and works after the code change.

Thanks.

**Comment by bryanlogan** [comment]
This got merged in via our private repo. We're now using invariant culture.

**Pull request closed without merging**